src/dashboards/videos/details.jsonnet

```diff
@@ -119,3 +120,88 @@ dashboard.new(
   ),
   gridPos={ x: 0, y: 0, w: 12, h: 9 }
 )
+.addPanel(
+  gaugePanel.new(
+    title='Views count by distinct sessions',
+    // It would be nice to use markdown inline code blocks but they appear as regular code blocks.
+    // This is fixed in v8.0.0-beta1 of grafana.
+    // See https://github.com/grafana/grafana/releases/tag/v8.0.0-beta1
+    description=|||
+      A view is counted when the user has started the video.
+
+      Note: We count a **view** only once in a session because the **video played** event
+      can be emitted multiple times while the video is played - for example:
+        - when we pause the video and then resume it
+        - when we pause the video, seek to the start and then resume it
+    |||,
+    datasource=lrs,
+    reducerFunction='sum',
+    unit='none'
+  ).addTarget(
+    elasticsearch.target(
+      datasource=lrs,
+      query='object.id.keyword:$VIDEO AND verb.id:"https://w3id.org/xapi/video/verbs/played"',
```

Comment:
I would rather use variables for field names, WDYT?

Reply:
Good Idea) Thanks) Updated using variables now)